docs/community/development.md

```diff
@@ -223,6 +223,9 @@ If the users have no way to know that things have changed, then you haven't real
 Likewise, if users don't know that there is a new feature that you've implemented, it will forever remain unknown and unused.
 
 The documentation has to be updated to users know that things have changed and added!
+If you are adding a new feature, add `(v<MERMAID_RELEASE_VERSION>+)` in the title or description. It will be replaced automatically with the current version number when the release happens.
+
+eg: `# Feature Name (v<MERMAID_RELEASE_VERSION>+)`
```

Comment:
Why do we change docs manually? Have we refused from CI that builds docs for us?

Reply:
This is for the contributors who add features. 

The replacement of MERMAID_RELEASE_VERSION will happen when the release manager runs `pnpm docs:release-version`. 

We removed the CI pushing changes in favour of a manual step.

Reply:
I would not remove the CI because it has some consistency, whereas local development setup does not, even `.lock` do not guarantee that. If someone compiles document it can be a little different every time. But I don't like a robot pushing to my branch either, because this is not obvious.